### PR TITLE
to utilize firstParagraphTransitions.txt when compiling essay

### PR DESCRIPTION
**Pull request by n8x0b7T**
You may want to check and see if this works before you make a new release or merge it.

### CODE REVIEW

**Comment by LittleCube-hax** [comment]
Ah, thank you.

Unfortunately I cannot test anything at the moment, there is a certain absurd bug that I have spent months after months trying to fix. The code is here in the repo, it's just not in the v1.4.2 release because, well, it doesn't work.

As soon as I get that up and running I'll let you know, and I'll test these changes as well.

**Comment by n8x0b7T** [comment]
Sounds good!

**Comment by n8x0b7T** [comment]
I also made some changes to outputwriter,java to actually use the txt file. I don't know if you saw that. It's in my fork of the project. My fork might be a bit out of date.

https://github.com/JandJ101/TheEssayWriter/blob/master/outputwriter.java

**Comment by LittleCube-hax** [comment]
It says your fork is 7 commits behind, maybe try pulling and see if the differences aren't too bad and maybe you can still merge?

**Comment by n8x0b7T** [comment]
i think i have it fixed now.
https://github.com/JandJ101/TheEssayWriter/blob/master/outputwriter.java

**Review by LittleCube-hax — CHANGES_REQUESTED** [review]
There are a few changes that need to be made, not huge, but they will give errors if left in.

File: outputwriter.java [hunk 1]
```diff
@@ -55,10 +56,21 @@ static String stripPunctuation(String str) {
 		}
 		return str;
 	}
-	
+
 	public static String createEssay() {
 		String essay = "";
 		try {
+
+      InputStream in = EssayWriter.class.getResourceAsStream("firstParagraphTransitions.txt");
```
Comment (LittleCube-hax) [inline outputwriter.java:64]: The file `firstParagraphTransitions.txt` doesn't exist.

File: outputwriter.java [hunk 2]
```diff
@@ -77,6 +90,10 @@ public static String createEssay() {
 				linee = buffer.readLine();
 			}
 			thirdTransitionWords = liness.toArray(new String[liness.size()]);
+
+      Random rand = new Random();
```
Comment (LittleCube-hax) [inline outputwriter.java:94]: If you initialize two Random objects (see line 97), they either have to be given different names, or you have to use the same object. Preferably the latter. So all you have to do (if I'm thinking correctly) is erase line 97 and leave your line 94 in.

**Pull request closed without merging**